Add docs for Application Management REST API

mefarazath opened the pull request:
This contains,
1. Link to postman collection
2. Application Swagger
3. Landing page for the application management REST API

ayshsandu commented on en/docs/develop/restapis/applications.yaml:
```diff
@@ -0,0 +1,2962 @@
+        authenticationSequence:
+          $ref: '#/components/schemas/AuthenticationSequence'
+        advancedConfigurations:
+          $ref: '#/components/schema
```
Are these ENUM values explained in respective endpoints?
If not, can you please explain all the enums used in the swagger definitions in relevant places.

mefarazath replied:
Explained the ENUMs in the AuthenticationSequence object

sherenewso2 commented on en/docs/develop/restapis/applications.yaml:
```diff
@@ -0,0 +1,2962 @@
+      operationId: getApplication
+      description: >
+        This API provides the capability to retrieve the application information
+        by id.
```
please refer to "id" as "ID" in all instances

sherenewso2 commented on en/docs/develop/restapis/applications.yaml:
```diff
@@ -0,0 +1,2962 @@
+      tags:
+        - Applications
+      summary: |
+        Create an application from an exported XML file.
```
Lets follow a similar format for all the "summary" fields. Since the first two follow this kind of format ("List applications, Add applications", shall we change this to "Create application from an exported XML file" ?

Please follow the same format in all the other summary fields without the full stop at the end.

sherenewso2 commented on en/docs/develop/restapis/applications.yaml:
```diff
@@ -0,0 +1,2962 @@
+        parameters of an application.
+
+        - There are three methods to create/update SAML2 authentication protocol configuration.
+            1. Metadata File (Need to send Base64 encoded content of the metadata file.)
```
Metadata file (by sending the Base64 encoded content of the metadata file.) 

Remove the fullstops after "Metadata URL" and "Manual configuration"

sherenewso2 commented on en/docs/develop/restapis/applications.yaml:
```diff
@@ -0,0 +1,2962 @@
+      summary: |
+        Regenerate the OAuth2/OIDC client secret.
+      description: |
+        This API provides regenerate the OAuth2/OIDC client secret.
```
This API provides **the capability to** regenerate the OAuth2/OIDC client secret.

or

This API regenerates the OAuth2/OIDC client secret.

sherenewso2 commented on en/docs/develop/restapis/applications.yaml:
```diff
@@ -0,0 +1,2962 @@
+      summary: |
+        Revokes the OAuth2/OIDC client.
+      description: |
+        This API provides revokes the OAuth2/OIDC client secret. To re-activate the client the client secret needs to be regenerated.
```
This API **revokes** the OAuth2/OIDC...

To reactivate the client **{comma}** the client secret....

sherenewso2 commented on en/docs/develop/restapis/applications.yaml:
```diff
@@ -0,0 +1,2962 @@
+        - Inbound Protocols - Custom
+      summary: >
+        Retrieve Custom Inbound authentication protocol parameters of an
+        application.
```
Custom --> custom

sherenewso2 replied:
please change all instances

sherenewso2 commented on en/docs/develop/restapis/applications.yaml:
```diff
@@ -0,0 +1,2962 @@
+        authenticationSequence:
+          $ref: '#/components/schemas/AuthenticationSequence'
+        advancedConfigurations:
+          $ref: '#/components/schema
```
Decides whether the application is accessible across tenants?

sherenewso2 replied:
Decides whether the list of authenticated identity providers need to **be** returned...

sherenewso2 replied:
**Decides whether** the refresh token....

mefarazath commented:
@ayshsandu @sherenewso2 I have addressed the comments and added the updated postman collection as well. Appreciate if you can have a look and approve :)
